<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="sideBar">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>控件盒子</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="MyToolButton" name="toolButton_Input">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Input</string>
         </property>
         <property name="icon">
          <iconset theme="mail-forward"/>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="MyToolButton" name="toolButton_Output">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Output</string>
         </property>
         <property name="icon">
          <iconset theme="mail-reply-all"/>
         </property>
         <property name="iconSize">
          <size>
           <width>35</width>
           <height>35</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="MyWidgetDraw" name="widget_Draw" native="true">
      <widget class="QWidget" name="widget_Page" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>681</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 245, 244);</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_NewProject"/>
    <addaction name="action_NewFile"/>
    <addaction name="action_OpenFile_Project"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_CloseAllFile"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_Back"/>
    <addaction name="action_Forward"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="action_Copy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_ChooseAll"/>
    <addaction name="separator"/>
    <addaction name="action_Delete"/>
    <addaction name="action_Delete2"/>
    <addaction name="separator"/>
    <addaction name="action_EditLabel"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="action_ToolBar"/>
    <addaction name="action_Object"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_NewProject_ToolBar"/>
   <addaction name="action_NewFile_ToolBar"/>
   <addaction name="action_OpenFile_Project_ToolBar"/>
   <addaction name="action_Save_ToolBar"/>
   <addaction name="separator"/>
   <addaction name="action_Options"/>
  </widget>
  <action name="action_NewProject">
   <property name="icon">
    <iconset theme="folder-new"/>
   </property>
   <property name="text">
    <string>新建项目</string>
   </property>
  </action>
  <action name="action_NewFile">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_OpenFile_Project">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>打开文件或项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存当前文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset theme="window-close"/>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出应用</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_CloseAllFile">
   <property name="icon">
    <iconset theme="edit-delete"/>
   </property>
   <property name="text">
    <string>关闭所有文件</string>
   </property>
  </action>
  <action name="action_Back">
   <property name="icon">
    <iconset theme="edit-undo"/>
   </property>
   <property name="text">
    <string>撤回</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Forward">
   <property name="icon">
    <iconset theme="edit-redo"/>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="icon">
    <iconset theme="edit-cut"/>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="icon">
    <iconset theme="edit-copy"/>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="icon">
    <iconset theme="edit-paste"/>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="icon">
    <iconset theme="edit-clear"/>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_ChooseAll">
   <property name="icon">
    <iconset theme="edit-select-all"/>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_EditLabel">
   <property name="icon">
    <iconset theme="computer"/>
   </property>
   <property name="text">
    <string>编辑标号</string>
   </property>
  </action>
  <action name="action_Delete2">
   <property name="icon">
    <iconset theme="edit-clear"/>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_NewProject_ToolBar">
   <property name="icon">
    <iconset theme="folder-new"/>
   </property>
   <property name="text">
    <string>新项目</string>
   </property>
   <property name="toolTip">
    <string>新建项目</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_OpenFile_Project_ToolBar">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开项目或文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save_ToolBar">
   <property name="icon">
    <iconset theme="document-save"/>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存当前文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_ToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>工具栏</string>
   </property>
   <property name="toolTip">
    <string>隐藏工具栏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_Object">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>控件栏</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="icon">
    <iconset theme="system-run"/>
   </property>
   <property name="text">
    <string>选项</string>
   </property>
   <property name="toolTip">
    <string>选择使用的模型和设置其他相关选项</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_NewFile_ToolBar">
   <property name="icon">
    <iconset theme="document-new"/>
   </property>
   <property name="text">
    <string>新文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyToolButton</class>
   <extends>QToolButton</extends>
   <header location="global">mytoolbutton.h</header>
  </customwidget>
  <customwidget>
   <class>MyWidgetDraw</class>
   <extends>QWidget</extends>
   <header location="global">mywidgetdraw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
